src/sandstorm/run-bundle.c++

```diff
@@ -1432,6 +1426,25 @@ private:
       tmpfsUidOpts = ",uid=0,gid=0";
     } else {
       unshareUidNamespaceOnce();
+
+      // In order to actually enter the new user namespace, we
```

Comment:
I don't think you need to fork to enter a userns. You do need to fork to enter a PID namespace.